examples/Advanced-Building-your-own-architecture.ipynb

```diff
@@ -0,0 +1,998 @@
+{
```

Comment:
<p>sugg: <code>built-in</code>.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/295/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='51'/>

Reply:
<p>Done</p>